src/launchpad/utils/__init__.py

```diff
@@ -1,6 +1,7 @@
 """Utility modules for app size analyzer."""
 
 from .android.bundletool import Bundletool, DeviceSpec
+from .console import console
```

Comment:
Not related to this change, but Trevor and I were chatting about the significance/usefulness of these __init__ exports. I know you have far more Python experience than us, if you know - what is the general pattern we should be following WRT exports in these files?